tools/clientconf.go

```diff
@@ -0,0 +1,187 @@
+package main
```

Comment:
Your current branch commits 2 files: `test_scripts/clientconf.go` and `tools/clientconf.go`

Reply:
fixed